### PR TITLE
Bladder dorsal ventral

### PR DESCRIPTION
**Pull request by zekh167**
Dorsal, ventral annotations and marker point were added to the bladderurethra1.py.

### CODE REVIEW

**Review by rchristie — CHANGES_REQUESTED** [review]

File: src/scaffoldmaker/annotation/bladder_terms.py [hunk 1]
```diff
@@ -4,19 +4,39 @@
...
+    ("urinary bladder", "UBERON:0001255", "FMA:15900"),
+    ("ventral part of bladder lumen", "None"),
+    ("ventral part of lumen of body of urinary bladder", "ILX:0739251"),
+    ("ventral part of lumen of neck of urinary bladder", "ILX:0739257"),
+    ("ventral part of lumen of urethra", "ILX:0739261"),
+    ("ventral part of serosa of body of urinary bladder", "ILX:0739279"),
+    ("ventral part of serosa of neck of urinary bladder", "ILX:0739281"),
+    ("ventral part of serosa of urethra", "ILX:0739306"),
+    ("ventral part of serosa of urinary bladder", "ILX:0739249"),
+    ("ventral part of the bladder", "None"),
```
Comment (rchristie) [inline src/scaffoldmaker/annotation/bladder_terms.py:38]: "ventral part of bladder".

File: src/scaffoldmaker/annotation/bladder_terms.py [hunk 2]
```diff
@@ -4,19 +4,39 @@
...
-    ("Dome of the Bladder", "ILX:0738433"),
-    ("serosa of urinary bladder", "UBERON:0001260"),
     ("bladder lumen", "UBERON:0009958"),
-    ("serosa of body of urinary bladder", "ILX:0739276"),
+    ("dome of the bladder", "ILX:0738433"),
+    ("dorsal part of bladder lumen", "None"),
+    ("dorsal part of lumen of body of urinary bladder", "ILX:0739250"),
+    ("dorsal part of lumen of neck of urinary bladder", "ILX:0739255"),
+    ("dorsal part of lumen of urethra", "ILX:0739260"),
+    ("dorsal part of the bladder", "None"),
```
Comment (rchristie) [inline src/scaffoldmaker/annotation/bladder_terms.py:13]: Rename to remove "the" --> "dorsal part of bladder" for consistency with other terms.
Also "ventral part of bladder" below.
(Exception: "dome of the bladder" is already in SciCrunch with "the", so can't change it).

**Review by rchristie — CHANGES_REQUESTED** [review]
Really minor name change then it's in.

File: src/scaffoldmaker/annotation/bladder_terms.py (hunk 2 above)
Comment (zekh167) [reply src/scaffoldmaker/annotation/bladder_terms.py]: done.

Comment (rchristie) [reply src/scaffoldmaker/annotation/bladder_terms.py]: Also update where they are used?

Comment (zekh167) [reply src/scaffoldmaker/annotation/bladder_terms.py]: Done Richard.

**Review by rchristie — APPROVED** [review]

**Pull request merged**